spec-main/api-web-contents-spec.ts

```diff
@@ -760,7 +761,7 @@ describe('webContents module', () => {
       }).to.throw('Must specify non-empty \'icon\' option');
 
       expect(() => {
-        w.webContents.startDrag({ file: __filename, icon: __filename });
+        w.webContents.startDrag({ file: __filename, icon: path.join(mainFixturesPath, 'blank.html') });
```

Comment:
yikes, it is surprising that specifying the js file would cause such a weird crash.

do we have any idea what the underlying cause is?

Reply:
@nornagon I'm wondering if it is because the file isn't an icon?  In my local testing I used an empty file and it was fine but when I used a file that had contents in it, it again failed.  This latest commit uses an empty file.

Reply:
Perhaps we could explicitly use an "empty file" file rather than overloading something that's expected in a lot of places to be html?

Reply:
@nornagon I figured out why this was crashing... it is an issue with NativeImage
![callstack3](https://user-images.githubusercontent.com/609052/90782711-2bd87300-e2cd-11ea-9b9c-8713f64b4ca2.png)
a `longjmp` is failing.  Apparently Windows thinks it is invalid for the process, as far as I can understand from this: https://docs.microsoft.com/en-us/windows/win32/devnotes/guardchecklongjumptarget

Here's where `setjmp` gets called: https://source.chromium.org/chromium/chromium/src/+/master:ui/gfx/codec/jpeg_codec.cc;l=179
and here's where `longjmp` gets called: https://source.chromium.org/chromium/chromium/src/+/master:ui/gfx/codec/jpeg_codec.cc;l=43